README.md

```diff
@@ -13,45 +13,170 @@ ZIO HTTP is a scala library for building http apps. It is powered by ZIO and [Ne
+- Portability: ZIO-HTTP applications are portable across different deployment platforms, making them versatile and adaptable.
+
+By leveraging the power of ZIO and the simplicity of functional programming, ZIO-HTTP provides a robust and flexible toolkit for building scalable and composable HTTP services in Scala.
+
+## Quickstart
+
+Eager to start coding without delay? If you're in a hurry, you can follow the [quickstart](https://github.com/zio/zio-http/tree/main/zio-http-example) guide or explore the [examples repository](https://github.com/zio/zio-http/tree/main/zio-http-example), which demonstrates different use cases and features of ZIO-HTTP.
+
+## Module feature overview
+
+Core:
+- Lightweight and performant HTTP handler and message objects
+- Powerful routing system with support for path-based and parameterized routes
+- Typesafe HTTP message construction and deconstruction
+- Extensible filters for common HTTP functionalities such as caching, compression, and request/response logging
+- Support for cookie handling
+- Servlet implementation for integration with Servlet containers
+- Built-in support for launching applications with an embedded server backend
+
+Client:
+
+- Robust and flexible HTTP client with support for synchronous and asynchronous operations
+- Adapters for popular HTTP client libraries such as Apache HttpClient, OkHttp, and Jetty HttpClient
+- Websocket client with blocking and non-blocking modes
+- GraphQL client integration for consuming GraphQL APIs
+
+Server:
+
+- Lightweight server backend spin-up for various platforms including Apache, Jetty, Netty, and SunHttp
+- Support for SSE (Server-Sent Events) and Websocket communication
+- Easy customization of underlying server backend
+- Native-friendly for compilation with GraalVM and Quarkus
+
+Serverless:
+
+- Function-based support for building serverless HTTP and event-driven applications
+- Adapters for AWS Lambda, Google Cloud Functions, Azure Functions, and other serverless platforms
+- Custom AWS Lambda runtime for improved performance and reduced startup time
+
+Contract:
 
-- Older library versions `1.x` or `2.x` with organization `io.d11` of ZIO Http are derived from Dream11, the organization that donated ZIO Http to the ZIO organization in 2022. 
-- Newer library versions, starting in 2023 and resulting from the ZIO organization (`dev.zio`) started with `0.0.x`, reaching `1.0.0` release candidates in April of 2023
+- Typesafe HTTP contract definition with support for path parameters, query parameters, headers, and request/response bodies
+- Automatic validation of incoming requests based on contract definition
+- Self-documenting routes with built-in support for OpenAPI (Swagger) descriptions
 
-## Getting Started
+Templating:
 
-A simple Http server can be built using a few lines of code.
+- Pluggable templating system support for popular template engines such as Dust, Freemarker, Handlebars, and Thymeleaf
```

Comment:
I see a lot of tools and technologies mentioned in this file that I doubt we currently support. Please make sure that they are related to ZIO‌ HTTP.

Reply:
@khajavi yea i intend on working on it  this week. what's up

Reply:
Great to hear that!